src/components/documentTable/index.tsx

```diff
@@ -66,6 +66,20 @@ export const DocumentsTable = (props: DocumentsTableProps) => {
     />
   ));
 
+  if (documents.length === 0) {
+    documentRows.push(
+      <TableRow>
+        <TableCell colSpan={10}>
+          <Grid container alignItems='center' justifyContent='center'>
+            <Typography variant='h5' color='grey'>
+              Nu exista rezultate
+            </Typography>
+          </Grid>
+        </TableCell>
+      </TableRow>
+    )
+  }
+
```

Comment:
Since we're using the same thing in two places, I think we can extract this generic row for empty tables into a separate component and use it where its needed.